src/components/camera.js

```diff
@@ -87,6 +88,30 @@ module.exports.Component = registerComponent('camera', {
     });
   },
 
+  /**
+   * Enables updating camera.userHeight
+   */
+  updateHeightOffset: function () {
+    var currentPosition;
+    var headsetConnected;
+    var el = this.el;
+    var oldHeightOffset = this.userHeightOffset;
```

Comment:
Should we get rid of `this.userHeightOffset`? This is always available at `this.data.userHeight` and the old height is available through `update (oldData)`, and we don't want extra state.